app/src/main/res/layout/contests_list_view.xml

```diff
@@ -12,23 +12,37 @@
     card_view:cardUseCompatPadding="true">
 
     <LinearLayout
-        style="@style/ListViewCardView"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical">
+        android:orientation="horizontal">
 
-        <TextView
-            android:id="@+id/tvContestName"
-            style="@style/HeaderText"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content" />
-
-        <TextView
-            android:id="@+id/tvContestTime"
-            android:layout_width="wrap_content"
+        <LinearLayout
+            style="@style/ListViewCardView"
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginTop="3dp"
-            android:textSize="12sp" />
+            android:layout_weight="1"
+            android:orientation="vertical">
+
+            <TextView
+                android:id="@+id/tvContestName"
+                style="@style/HeaderText"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content" />
 
+            <TextView
+                android:id="@+id/tvContestTime"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="3dp"
+                android:textSize="12sp" />
+
+        </LinearLayout>
+
+        <ImageView
```

Comment:
Well, one day I'll create a design in Sketch for Codeforces Watcher too, but not today. I'll try to explain in plain words for now:
- all (almost) icons in mobile apps should be of the same size: 24dp x 24dp
- margins should be consistent (here you can add end margin of `16dp` and center `ImageView` vertically
- colors of icons should be consistent too (use the color of unselected icon from bottom nav bar)